test/functionalTest/cases/1426_empty_components_in_uri_path/double_slash_at_end_of_path.test

```diff
@@ -0,0 +1,566 @@
+echo
+echo
+
+
+echo "04a. /v2/entities/E1/attrs//"
+echo "============================"
+orionCurl --url /v2/entities/E1/attrs//
+echo
+echo
+
+
+echo "04b. /v2/entities/E1/attrs///"
+echo "============================="
+orionCurl --url /v2/entities/E1/attrs///
+echo
+echo
+
+
+echo "05a. /v2/entities/E1/attrs/A1//"
+echo "==============================="
+orionCurl --url /v2/entities/E1/attrs/A1//
+echo
+echo
+
+
+echo "05b. /v2/entities/E1/attrs/A1///"
+echo "================================"
+orionCurl --url /v2/entities/E1/attrs/A1///
+echo
+echo
+
+
+echo "06a. /v2/entities/E1/attrs/A1/value//"
+echo "====================================="
+orionCurl --url /v2/entities/E1/attrs/A1/value//
+echo
+echo
+
+
+echo "06b. /v2/entities/E1/attrs/A1/value///"
+echo "======================================"
+orionCurl --url /v2/entities/E1/attrs/A1/value///
+echo
+echo
+
+
+echo "07a. /v2/types//"
+echo "================"
+orionCurl --url /v2/types//
+echo
+echo
+
+
+echo "07b. /v2/types///"
+echo "================="
+orionCurl --url /v2/types///
+echo
+echo
+
+
+echo "08a. /v2/types/T1//"
+echo "==================="
+orionCurl --url /v2/types/T1//
+echo
+echo
+
+
+echo "08b. /v2/types/T1///"
+echo "===================="
+orionCurl --url /v2/types/T1///
+echo
+echo
+
+
+echo "09a. /v2/subscriptions//"
+echo "========================"
+orionCurl --url /v2/subscriptions//
+echo
+echo
+
+
+echo "09b. /v2/subscriptions///"
+echo "========================="
+orionCurl --url /v2/subscriptions///
+echo
+echo
+
+
+echo "10a. /v2/subscriptions/subId//"
+echo "=============================="
+orionCurl --url /v2/subscriptions/subId//
+echo
+echo
+
+
+echo "10b. /v2/subscriptions/subId///"
+echo "==============================="
+orionCurl --url /v2/subscriptions/subId///
+echo
+echo
+
+
+echo "11a. /v2/op/query//"
+echo "==================="
+orionCurl --url /v2/op/query//
+echo
+echo
+
+
+echo "11b. /v2/op/query///"
+echo "===================="
+orionCurl --url /v2/op/query///
+echo
+echo
+
+
+echo "12a. /v2/op/update//"
+echo "===================="
+orionCurl --url /v2/op/update//
+echo
+echo
+
+
+echo "12b. /v2/op/update///"
+echo "====================="
+orionCurl --url /v2/op/update///
+echo
+echo
+
+
+--REGEXPECT--
+01a. /v2//
+==========
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+01b. /v2///
+===========
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+02a. /v2/entities//
+===================
+HTTP/1.1 404 Not Found
+Content-Length: 95
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "The requested entity has not been found. Check type and id",
```

Comment:
Some test must be missing somewhere.
Nothing to do with this PR, but, what the heck, I can try to find the error within the PR.